### ISSUE TITLE
Render Thread Client Crash (Hugh Modpack to test Compat issues) Fabric 1.21.1 on 0.16.9

### ISSUE THREAD

**Issue by suerion** [Alminoris/ArborealNature#1]
Hey, i had searched long about this issue, allways with aboreal Nature added, the client Crash:

https://mclo.gs/kmy69Cd

had last time thought it was supplementaries, but it is aboreal-nature.

Hope you could fix this issue

https://github.com/MehVahdJukaar/Supplementaries/issues/1196#event-15196925722

**Comment by Alminoris** [comment]
As I understand it, the problem occurs when interacting with the dispenser, specifically its GUI Screen. Did you use items, blocks from Arboreal Nature mod in the dispenser? If yes, which ones (maybe it's a matter of tags)? If no, then maybe it's in Supplementaries (because you use not the latest version, but alpha 3.0.6, try a newer version). In my mod there are no blocks with GUI, and the problem is related to this.

**Comment by suerion** [comment]
Hey @Alminoris it crash on joining the Server, it does not crash on Playing. I had no Dispenser or Something.

**Comment by Alminoris** [comment]
Sorry, I just thought the problem was on the integrated server. I've never tested my mod on the server yet. And when I tested it, there are no problems, but when I tried Supplementaries as well, it gives these errors: ‘Couldn't parse element minecraft:loot_table/supplementaries:blocks/urn_loot/urn_loot’.
Couldn't parse element minecraft:loot_table/supplementaries:blocks/urn_loot/epic
’, ‘Unknown registry key in ResourceKey[minecraft:root / minecraft:loot_function_type]: supplementaries:random_enchantment
’, ‘Failed to parse resource conditions for file of type recipe with id supplementaries:cabin_siding
’, ‘Unknown loot table called supplementaries:blocks/urn_loot/urn_loot
‘. And they have to do exclusively with the Supplementaries mod.

**Comment by suerion** [comment]
Then you have here an new issue on Server side. I had disabled all Mods and tried to use it without and with supplementaries. The Crash occurs on your mod. I thought in the first time, i had an issue with supplementary, but after testing and disabling mods and adding mods, it gives me your mod as Culprit on the joining of the Server. Without, it does not crash. 

It is client side crash on joining Server side.

**Comment by Alminoris** [comment]
I've already figured out that duplicate IDs for signs and hanging signs are to blame here. I think it's an issue with registration, but I haven't been able to resolve it yet. I'm using the Terraform Wood API to register the signs.

**Comment by suerion** [comment]
Hey, you could look on terralith, sawmill etc to look on wood

sorry i had meaned https://www.curseforge.com/minecraft/mc-mods/terrablender-fabric

**Comment by suerion** [comment]
@Alminoris the latest version is working for me now. Can join server and so on. If i got new issues, i will be come back here

**Comment by suerion** [comment]
had got this issue 
```
[16:12:06] [Render thread/WARN]: Unable to load model: 'arborealnature:item/bilberry_bush' referenced from: arborealnature:item/bilberry_bush: com.google.common.cache.CacheLoader$InvalidCacheLoadException: CacheLoader returned null for key arborealnature:models/item/bilberry_bush.json.

[16:12:06] [Render thread/WARN]: Unable to load model: 'arborealnature:item/blackberry_bush' referenced from: arborealnature:item/blackberry_bush: com.google.common.cache.CacheLoader$InvalidCacheLoadException: CacheLoader returned null for key arborealnature:models/item/blackberry_bush.json.

[16:12:06] [Render thread/WARN]: Unable to load model: 'arborealnature:item/pink_currant_bush' referenced from: arborealnature:item/pink_currant_bush: com.google.common.cache.CacheLoader$InvalidCacheLoadException: CacheLoader returned null for key arborealnature:models/item/pink_currant_bush.json.

[16:12:28] [Render thread/WARN]: Unable to load model: 'arborealnature:item/bilberry_bush' referenced from: arborealnature:item/bilberry_bush: com.google.common.cache.CacheLoader$InvalidCacheLoadException: CacheLoader returned null for key arborealnature:models/item/bilberry_bush.json.

[16:12:28] [Render thread/WARN]: Unable to load model: 'arborealnature:item/blackberry_bush' referenced from: arborealnature:item/blackberry_bush: com.google.common.cache.CacheLoader$InvalidCacheLoadException: CacheLoader returned null for key arborealnature:models/item/blackberry_bush.json.

[16:12:28] [Render thread/WARN]: Unable to load model: 'arborealnature:item/pink_currant_bush' referenced from: arborealnature:item/pink_currant_bush: com.google.common.cache.CacheLoader$InvalidCacheLoadException: CacheLoader returned null for key arborealnature:models/item/pink_currant_bush.json.
```

**Comment by Alminoris** [comment]
Just released a 0.2.7.1 patch with fixes for issue with "Unable to load model" for berry bushes.

**Comment by suerion** [comment]
I think it worked fine now
